### PR TITLE
chore: Generate CDN in init-container.sh 🦭

### PR DESCRIPTION
**Pull request by darcywong00**
Continuing #337

* Fix cdn pathing for Docker container
* Generate CDN in init-container.sh

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -38,8 +38,16 @@ function test_docker_container() {
   echo "Done checking links"
 }
 
+# Custom cleanup of CDN
+function clean_docker_container_cdn() {
+  clean_docker_container $KEYMAN_IMAGE_NAME $KEYMAN_CONTAINER_NAME
+
+  # Cleanup CDN
+  rm -rf cdn/deploy
```
Comment (mcdurdin) [inline build.sh:46]: We probably don't need to clean this up -- it's only a cache so doesn't break things if it is present.

File: cdn/cdnrefresh.php [hunk 2]
```diff
@@ -36,7 +36,7 @@ function generate_hashed_version_file($source, $deploy, $source_file) {
       $dest_file = $source_file_short;
     }
     if(!file_exists($deploy . $dest_file)) {
-      echo $source_file . " => " . $deploy . $dest_file . "\n";
+      echo $source_file . " => " . $deploy . $dest_file . "<br/>";
```
Comment (mcdurdin) [inline cdn/cdnrefresh.php:39]: This is a CLI process, why emit html?

**Review by mcdurdin — APPROVED** [review]
LGTM apart from those two nits :grin:

File: cdn/cdnrefresh.php (hunk 2 above)
Comment (darcywong00) [reply cdn/cdnrefresh.php:39]: I used https://github.com/keymanapp/keymanweb.com/blob/0dba6472d3300ef15a4820ee53953d0051333aa0/cdn/cdnrefresh.php#L39 as reference

Comment (darcywong00) [reply cdn/cdnrefresh.php]: Reverted to "\n"

Comment (mcdurdin) [reply cdn/cdnrefresh.php]: gotcha

**Pull request merged**